renderdoc/driver/vulkan/wrappers/vk_cmd_funcs.cpp

```diff
@@ -7570,6 +7570,36 @@ void WrappedVulkan::vkCmdEndRendering(VkCommandBuffer commandBuffer)
   }
 }
 
+VkResult WrappedVulkan::vkCreateDeferredOperationKHR(VkDevice device,
```

Comment:
This is definitely nitpicky but these shouldn't go in `vk_cmd_funcs.cpp` as they're not vkCmd* functions.

The categorisation is definitely loosey-goosey so there's not a clear place. I'd say _maybe_ `vk_sync_funcs.cpp` but probably just `vk_misc_funcs.cpp`. The two `vkGet*` functions could go in `vk_get_funcs.cpp` but it'd also be fine to leave them here.

Reply:
I moved the non-Get funcs into `vk_misc_funcs.cpp` and the Get ones into `vk_get_funcs.cpp`.